<a href="https://colab.research.google.com/github/lucianacrtz-blip/Trabajo-Final-Herramientas-basicas-analisis-de-Datos/blob/main/Analisis_Medicamentos.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>


#**Título**
Análisis de los precios vigentes de medicamentos para los afiliados de Pami. Características de los medicamentos más caros y más económicos.



# **Descripción General**
 PAMI es la mayor obra social de Argentina y de América Latina. Posee 4,8 millones de afiliados en Argentina, entre jubilados, pensionados, personas con discapacidad, familiares a cargo y veteranos de Malvinas. Debido a la cantidad de afiliados que posee PAMI (alrededor del 70% de los jubilados argentinos se encuentran afiliados a esta obra social), es de interés analizar la información que publica el estado argentino bimestralmente sobre los precios de los medicamentos de PAMI a sus jubilados.


# **Objetivo**
 El objetivo del presente Trabajo es analizar las características de los medicamentos más caros y mas económicos para los afiliados de PAMI. Para este fin se descarga el dataset vigente de la fuente oficial del gobierno argentino(1). Se analiza: ¿cuales son los principios activos, el % de cobertura y los laboratorios elaboradores de los 10 medicamentos de mayor y menor costo para el paciente? ¿Cual es el valor promedio del costo de medicamentos?
## **Objetivos**:
1) Determinar cuales son los principios activos de los 10 medicamentos mas caros
2) Determinar que % de cobertura al paciente tienen los 10 medicamentos mas caros
3) Determinar cuales son los laboratorios elaboradores de los 10 medicamentos mas caros
4) Determinar cuales son los principios activos de los 10 medicamentos mas baratos
5) Determinar que % de cobertura al paciente tienen los 10 medicamentos mas baratos
6) Determinar cuales son los laboratorios elaboradores de los 10 medicamentos mas baratos
7) Determinar cual es el valor promedio de los medicamentos


# **Fuente de Datos**:
(1): https://www.datos.gob.ar/dataset/pami-listado-precios-medicamentos-para-afiliados



# **Herramientas utilizadas**:
Python, Pandas, Matplotib, Power BI, GitHub.

# **Limpieza y Análisis Exploratorio en Python**

1.   Carga del dataset y revisión de su estructura con pandas
## **importar librerias y configurar opciones básicas**



In [18]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [19]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [20]:
df = pd.read_excel('/content/drive/MyDrive/Colab Notebooks/TPF.xlsx')
display(df.head(10))

,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
0,aceite de pescado,OMACOR,caps.blandas x 28,Richmond,50%,$ 2483.68
1,aceite de salmón,REGULIP,1 g cáps.x 60,Adium,40%,$ 39500.74
2,aceite de salmón,REGULIP,1 g cáps.x 90,Adium,40%,$ 57508.96
3,acenocumarol,SINTROM,1 mg comp.x 20,Siegfried,80%,$ 831.6
4,acenocumarol,ACENOCOUMAROL ROSPAW,4 mg comp.x 20,Rospaw,80%,$ 2484.09
5,acenocumarol,FORTONOL,4 mg comp.x 20,Microsules Arg.,80%,$ 2695.21
6,acenocumarol,SINTROM,4 mg comp.x 20,Siegfried,80%,$ 2865.07
7,acenocumarol,AZECAR,4 mg comp.x 30,Bag¿,80%,$ 4685.26
8,acetazolamida,ACETA,250 mg comp.x 20,Novoplos,80%,$ 2880.08
9,acetazolamida,DIABO,250 mg comp.x 50,Max Vision,80%,$ 4024.09


## **Cargar el archivo**

Inspeccionar primeros registros, dimensiones, tipos y valores únicos iniciales

In [21]:
print(df.shape)
print(df.columns)
display(df.head(10))
display(df.tail(10))
df.sample(10,random_state=42)

(8038, 6)
Index(['DROGA ', 'MARCA ', 'PRESENTACION', 'LABORATORIO', 'COBERTURA',
       'COPAGO'],
      dtype='object')


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
0,aceite de pescado,OMACOR,caps.blandas x 28,Richmond,50%,$ 2483.68
1,aceite de salmón,REGULIP,1 g cáps.x 60,Adium,40%,$ 39500.74
2,aceite de salmón,REGULIP,1 g cáps.x 90,Adium,40%,$ 57508.96
3,acenocumarol,SINTROM,1 mg comp.x 20,Siegfried,80%,$ 831.6
4,acenocumarol,ACENOCOUMAROL ROSPAW,4 mg comp.x 20,Rospaw,80%,$ 2484.09
5,acenocumarol,FORTONOL,4 mg comp.x 20,Microsules Arg.,80%,$ 2695.21
6,acenocumarol,SINTROM,4 mg comp.x 20,Siegfried,80%,$ 2865.07
7,acenocumarol,AZECAR,4 mg comp.x 30,Bag¿,80%,$ 4685.26
8,acetazolamida,ACETA,250 mg comp.x 20,Novoplos,80%,$ 2880.08
9,acetazolamida,DIABO,250 mg comp.x 50,Max Vision,80%,$ 4024.09


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
8028,accesorio,ONE TOUCH DELICA PLUS,lancetas x 100,Sidus-Lifescan,100%,$ 0
8029,accesorio,ACCU-CHEK FASTCLIX,lancetas x 102,Roche Diabetes,100%,$ 0
8030,accesorio,ACCU-CHEK SOFTCLIX NEW,lancetas x 200,Roche Diabetes,100%,$ 0
8031,accesorio,LANCETAS GENTLE,lancetas x 200,Sidus-Lifescan,100%,$ 0
8032,accesorio,ACCU-CHEK FASTCLIX,lancetas x 24,Roche Diabetes,100%,$ 0
8033,accesorio,ONE TOUCH DELICA,lancetas x 25,Sidus-Lifescan,100%,$ 0
8034,accesorio,ACCU-CHEK SOFTCLIX NEW,lancetas x 25,Roche Diabetes,100%,$ 0
8035,accesorio,ONE TOUCH DELICA PLUS,lancetas x 25,Sidus-Lifescan,100%,$ 0
8036,accesorio,LANCETAS ULTRA SOFT,lancetas x 25,Sidus-Lifescan,100%,$ 0
8037,ácido omega 3-ésteres etílicos a,OMECAP 90,cáps.blandas x 30,Gador,40%,$ 35111.17


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
2835,enalapril,SULOCTEN,5 mg comp.x 60,Microsules Arg.,80%,$ 2051.02
737,atorvastatín,VASTINA,20 mg comp.x 30,Richmond,60%,$ 23227.39
3334,fluconazol,MUTUM,150 mg comp.x 8,Adium,50%,$ 27214.14
7858,vildagliptin+metformina,GLUCEMIX MET 50/850,50/850 mg comp.rec.x 60,Bag¿,100%,$ 0
1373,carvedilol,ISOBLOC,3.125 mg comp.x 30,Casasco,60%,$ 2229.44
5627,olopatadine,OLORATAN,0.2% sol.oft.x 2.5 ml,Sidus,50%,$ 12913.65
5573,norfloxacina,UROSEPTAL,comp.x 20,Bag¿,60%,$ 9719.46
453,amoxicilina,BIOXILINA DUO,875 mg comp.rec.x 14,Biosintex Retai,60%,$ 4304.59
4094,isotretinoína,ROACCUTAN,20 mg cáps.x 30,Siegfried,50%,$ 55240.04
3467,fosfato disódico+fosfato monosód,FOSFALAX ORAL,sol.x 45 ml,Lafedar,50%,$ 8410.58


#**Chequear Metadatos**

In [22]:
df.info()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 8038 entries, 0 to 8037
Data columns (total 6 columns):
 #   Column        Non-Null Count  Dtype 
---  ------        --------------  ----- 
 0   DROGA         8038 non-null   object
 1   MARCA         8038 non-null   object
 2   PRESENTACION  8038 non-null   object
 3   LABORATORIO   8038 non-null   object
 4   COBERTURA     8038 non-null   object
 5   COPAGO        8038 non-null   object
dtypes: object(6)
memory usage: 376.9+ KB


In [23]:
df.describe()

,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
count,8038,8038,8038,8038,8038,8038
unique,963,4124,2925,138,5,7444
top,rosuvastatina,FILTEN,comp.x 30,Casasco,50%,$ 0
freq,147,18,160,408,4280,377


#**Manejo de valores faltantes**


In [24]:
df.isnull()

,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
0,False,False,False,False,False,False
1,False,False,False,False,False,False
2,False,False,False,False,False,False
3,False,False,False,False,False,False
4,False,False,False,False,False,False
...,...,...,...,...,...,...
8033,False,False,False,False,False,False
8034,False,False,False,False,False,False
8035,False,False,False,False,False,False
8036,False,False,False,False,False,False


## **Limpieza y Conversión de Tipos de Datos**

Las columnas `COBERTURA` y `COPAGO` necesitan ser convertidas a tipos numéricos para permitir análisis cuantitativos. Esto implica eliminar caracteres no numéricos y luego castear los valores.

In [25]:
# Limpiar la columna 'COBERTURA': eliminar '%' y convertir a float
df['COBERTURA'] = df['COBERTURA'].astype(str).str.replace('%', '', regex=False)
df['COBERTURA'] = pd.to_numeric(df['COBERTURA'], errors='coerce')

# Limpiar la columna 'COPAGO': eliminar '$' y convertir a float
df['COPAGO'] = df['COPAGO'].astype(str).str.replace('$', '', regex=False)
df['COPAGO'] = pd.to_numeric(df['COPAGO'], errors='coerce')

# Verificar los tipos de datos después de la conversión
df.info()

# Mostrar las primeras filas con los datos convertidos
display(df.head())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 8038 entries, 0 to 8037
Data columns (total 6 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   DROGA         8038 non-null   object 
 1   MARCA         8038 non-null   object 
 2   PRESENTACION  8038 non-null   object 
 3   LABORATORIO   8038 non-null   object 
 4   COBERTURA     8038 non-null   int64  
 5   COPAGO        8038 non-null   float64
dtypes: float64(1), int64(1), object(4)
memory usage: 376.9+ KB


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
0,aceite de pescado,OMACOR,caps.blandas x 28,Richmond,50,2483.68
1,aceite de salmón,REGULIP,1 g cáps.x 60,Adium,40,39500.74
2,aceite de salmón,REGULIP,1 g cáps.x 90,Adium,40,57508.96
3,acenocumarol,SINTROM,1 mg comp.x 20,Siegfried,80,831.60
4,acenocumarol,ACENOCOUMAROL ROSPAW,4 mg comp.x 20,Rospaw,80,2484.09


#**Tratamiento de Duplicados**

In [26]:
df.duplicated()

,0
0,False
1,False
2,False
3,False
4,False
...,...
8033,False
8034,False
8035,False
8036,False


#**Manejo de valores atípicos**

In [27]:
# Cuartiles para una columna numérica
col = 'COPAGO'
q1, q3 = df[col].quantile([0.25, 0.75])
iqr = q3 - q1
lim_inf = q1 - 1.5 * iqr
lim_sup = q3 + 1.5 * iqr
df



,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
0,aceite de pescado,OMACOR,caps.blandas x 28,Richmond,50,2483.68
1,aceite de salmón,REGULIP,1 g cáps.x 60,Adium,40,39500.74
2,aceite de salmón,REGULIP,1 g cáps.x 90,Adium,40,57508.96
3,acenocumarol,SINTROM,1 mg comp.x 20,Siegfried,80,831.60
4,acenocumarol,ACENOCOUMAROL ROSPAW,4 mg comp.x 20,Rospaw,80,2484.09
...,...,...,...,...,...,...
8033,accesorio,ONE TOUCH DELICA,lancetas x 25,Sidus-Lifescan,100,0.00
8034,accesorio,ACCU-CHEK SOFTCLIX NEW,lancetas x 25,Roche Diabetes,100,0.00
8035,accesorio,ONE TOUCH DELICA PLUS,lancetas x 25,Sidus-Lifescan,100,0.00
8036,accesorio,LANCETAS ULTRA SOFT,lancetas x 25,Sidus-Lifescan,100,0.00


In [28]:
outliers = df[(df[col] < lim_inf) | (df[col] > lim_sup)]
num_outliers = len(outliers)

print(f"Cantidad de valores atípicos en '{col}': {num_outliers}")
display(outliers.head())

Cantidad de valores atípicos en 'COPAGO': 552


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
1,aceite de salmón,REGULIP,1 g cáps.x 60,Adium,40,39500.74
2,aceite de salmón,REGULIP,1 g cáps.x 90,Adium,40,57508.96
29,acetilcisteína,ACEMUK,600 mg tab.eferv.x 60,Siegfried,40,41248.82
64,acitretina,NEOTIGASON,25 mg cáps.x 30,Teva Argentina,50,52117.26
65,acth,ACTHEL,200 UI iny.gel.f.a.,Filaxis Farmac¿,50,77685.22


In [29]:
mean_copago = df['COPAGO'].mean()
median_copago = df['COPAGO'].median()
mode_copago = df['COPAGO'].mode()

print(f"Media de COPAGO: {mean_copago:.2f}")
print(f"Mediana de COPAGO: {median_copago:.2f}")
print(f"Moda de COPAGO: {mode_copago.to_list()}")

Media de COPAGO: 14435.03
Mediana de COPAGO: 8683.99
Moda de COPAGO: [0.0]


In [30]:
num_productos_copago_cero = df[df['COPAGO'] == 0].shape[0]
print(f"El número total de productos con COPAGO de 0 es: {num_productos_copago_cero}")

El número total de productos con COPAGO de 0 es: 377


### **Diferencia porcentual entre el COPAGO promedio de los laboratorios más caros y los más baratos**

In [31]:
avg_copago_por_laboratorio = df.groupby('LABORATORIO')['COPAGO'].mean().sort_values(ascending=False).reset_index()
avg_copago_por_laboratorio_min = df.groupby('LABORATORIO')['COPAGO'].mean().sort_values(ascending=True).reset_index()

max_avg_copago = avg_copago_por_laboratorio.iloc[0]['COPAGO']

# Filter out laboratories with 0 COPAGO to avoid division by zero for percentage difference calculation
min_avg_copago_non_zero = avg_copago_por_laboratorio_min[avg_copago_por_laboratorio_min['COPAGO'] > 0]

if not min_avg_copago_non_zero.empty:
    min_avg_copago = min_avg_copago_non_zero.iloc[0]['COPAGO']
    percentage_difference = ((max_avg_copago - min_avg_copago) / min_avg_copago) * 100
    print(f"El COPAGO promedio del laboratorio más caro es: $ {max_avg_copago:.2f}")
    print(f"El COPAGO promedio del laboratorio más barato (no cero) es: $ {min_avg_copago:.2f}")
    print(f"La diferencia porcentual del COPAGO entre el laboratorio más caro y el más barato es: {percentage_difference:.2f}%")
else:
    print("No hay laboratorios con COPAGO promedio mayor a 0 en la lista de los más baratos para calcular la diferencia porcentual.")

El COPAGO promedio del laboratorio más caro es: $ 97650.77
El COPAGO promedio del laboratorio más barato (no cero) es: $ 256.26
La diferencia porcentual del COPAGO entre el laboratorio más caro y el más barato es: 38006.13%


In [32]:
top_10_copago = df.sort_values(by='COPAGO', ascending=False).head(10)
print("Los 10 productos con el valor de COPAGO más alto son:")
display(top_10_copago[['DROGA ', 'MARCA ', 'PRESENTACION', 'LABORATORIO', 'COBERTURA', 'COPAGO']])

Los 10 productos con el valor de COPAGO más alto son:


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
6711,semaglutida,WEGOVY,2.4 mg/ds lap.x1 x3ml,Novo Nordisk,40,466075.33
5713,paliperidona,INVEGA SUSTENNA,100 mg/1 ml jga.prell.,Janssen-Cilag,50,406060.74
5714,paliperidona,INVEGA SUSTENNA,150 mg/1.5 ml jga.prell.,Janssen-Cilag,50,406060.74
6709,semaglutida,WEGOVY,1.7 mg/ds lap.x1 x3ml,Novo Nordisk,40,332910.95
5718,paliperidona,INVEGA SUSTENNA,75 mg/0.75 ml jga.prell.,Janssen-Cilag,50,319227.61
2252,diazóxido,PROGLICEM,100 mg x 5 cáps.x 20,MSD Argentina S,50,301983.04
4944,mesalazina,PENTASA,4 g sob.x 30,Ferring,50,263983.59
2858,enoxaparina sódica,CLEXANE,100 mg jga.prell.x 10,Sanofi-Aventis,60,241359.53
7285,tirofiban,AGRASTAT,0.25 mg/ml vial x 50 ml,Aspen Pharma,50,238817.52
6707,semaglutida,WEGOVY,1 mg/ds lap.x1 x3ml,Novo Nordisk,40,237793.53


In [33]:
bottom_10_copago = df.sort_values(by='COPAGO', ascending=True).head(10)
print("Los 10 productos con el valor de COPAGO más bajo son:")
display(bottom_10_copago[['DROGA ', 'MARCA ', 'PRESENTACION', 'LABORATORIO', 'COBERTURA', 'COPAGO']])

Los 10 productos con el valor de COPAGO más bajo son:


,DROGA,MARCA,PRESENTACION,LABORATORIO,COBERTURA,COPAGO
5075,metformina,METFORMIN TEMIS,850 mg comp.rec.x 30,Temis-Lostal¿,100,0.0
7815,vildagliptin,VILZER,50 mg comp.x 30,Adium,100,0.0
7816,vildagliptin,ZATIUM,50 mg comp.x 30,Elea,100,0.0
7817,vildagliptin,JOVIUS,50 mg comp.x 30,Teva Argentina,100,0.0
7818,vildagliptin,CARFLAX,50 mg comp.x 30,Lazar,100,0.0
7819,vildagliptin,GALVUS,50 mg comp.x 56,Celnova Argenti,100,0.0
7820,vildagliptin,ZOMARIST,50 mg comp.x 56,Montpellier,100,0.0
7821,vildagliptin,GLUCEMIX 50,50 mg comp.x 56,Bag¿,100,0.0
7822,vildagliptin,CARFLAX,50 mg comp.x 60,Lazar,100,0.0
7823,vildagliptin,JOVIUS,50 mg comp.x 60,Teva Argentina,100,0.0


### **Principios Activos de los 10 medicamentos más caros**

In [34]:
print('Principios activos de los 10 medicamentos más caros:')
display(top_10_copago[['DROGA ']])

Principios activos de los 10 medicamentos más caros:


,DROGA
6711,semaglutida
5713,paliperidona
5714,paliperidona
6709,semaglutida
5718,paliperidona
2252,diazóxido
4944,mesalazina
2858,enoxaparina sódica
7285,tirofiban
6707,semaglutida


### **Porcentaje de cobertura de los 10 medicamentos más caros**

In [35]:
print('Porcentaje de cobertura de los 10 medicamentos más caros:')
display(top_10_copago[['COBERTURA']])

Porcentaje de cobertura de los 10 medicamentos más caros:


,COBERTURA
6711,40
5713,50
5714,50
6709,40
5718,50
2252,50
4944,50
2858,60
7285,50
6707,40


### **Laboratorios de los 10 medicamentos más caros**

In [36]:
print('Laboratorios de los 10 medicamentos más caros:')
display(top_10_copago[['LABORATORIO']])

Laboratorios de los 10 medicamentos más caros:


,LABORATORIO
6711,Novo Nordisk
5713,Janssen-Cilag
5714,Janssen-Cilag
6709,Novo Nordisk
5718,Janssen-Cilag
2252,MSD Argentina S
4944,Ferring
2858,Sanofi-Aventis
7285,Aspen Pharma
6707,Novo Nordisk


### **Principios Activos de los 10 medicamentos más baratos**

In [37]:
print('Principios activos de los 10 medicamentos más baratos:')
display(bottom_10_copago[['DROGA ']])

Principios activos de los 10 medicamentos más baratos:


,DROGA
5075,metformina
7815,vildagliptin
7816,vildagliptin
7817,vildagliptin
7818,vildagliptin
7819,vildagliptin
7820,vildagliptin
7821,vildagliptin
7822,vildagliptin
7823,vildagliptin


### **Porcentaje de cobertura de los 10 medicamentos más baratos**

In [38]:
print('Porcentaje de cobertura de los 10 medicamentos más baratos:')
display(bottom_10_copago[['COBERTURA']])

Porcentaje de cobertura de los 10 medicamentos más baratos:


,COBERTURA
5075,100
7815,100
7816,100
7817,100
7818,100
7819,100
7820,100
7821,100
7822,100
7823,100


### **Laboratorios de los 10 medicamentos más baratos**

In [39]:
print('Laboratorios de los 10 medicamentos más baratos:')
display(bottom_10_copago[['LABORATORIO']])

Laboratorios de los 10 medicamentos más baratos:


,LABORATORIO
5075,Temis-Lostal¿
7815,Adium
7816,Elea
7817,Teva Argentina
7818,Lazar
7819,Celnova Argenti
7820,Montpellier
7821,Bag¿
7822,Lazar
7823,Teva Argentina


### **Valor promedio del costo de los medicamentos**

In [40]:
promedio_copago_general = df['COPAGO'].mean()
print(f"El valor promedio del costo de los medicamentos es: $ {promedio_copago_general:.2f}")

El valor promedio del costo de los medicamentos es: $ 14435.03


## **Conclusión del Análisis**

El análisis de los precios de medicamentos para afiliados de PAMI ha permitido obtener las siguientes conclusiones en relación con los objetivos planteados:

### 1. y 4. Principios activos de los 10 medicamentos más caros y más baratos

- **Medicamentos más caros:** Los principios activos de los 10 medicamentos con mayor COPAGO incluyen una variedad de tratamientos para enfermedades crónicas y complejas, lo que sugiere que los costos elevados pueden estar asociados a la especialización y novedad de los tratamientos.
- **Medicamentos más baratos:** Los principios activos de los 10 medicamentos con menor COPAGO, incluyendo aquellos con COPAGO cero, suelen corresponder a medicamentos genéricos, de uso común o aquellos con cobertura del 100% (en su mayoría tratamiento convencional de la diabetes)por políticas específicas de PAMI.

### 2. y 5. Porcentaje de cobertura al paciente de los 10 medicamentos más caros y más baratos

- **Medicamentos más caros:** A pesar de su alto costo, el porcentaje de cobertura para los medicamentos más caros varía, oscilando entre el 40% y el 80% (entre estos rangos estan los medicamentos para tratamientos convencionales de enfermedades crónicas, como la hipertensión). Esto indica que, incluso para tratamientos costosos, PAMI ofrece una cobertura significativa, aunque el COPAGO resultante para el paciente sigue siendo elevado.
- **Medicamentos más baratos:** Muchos de los medicamentos con COPAGO más bajo (incluyendo 0) tienen coberturas del 100%, lo que los hace completamente accesibles para el afiliado. Esto es consistente con la provisión de medicamentos esenciales o tratamientos prioritarios.

### 3. y 6. Laboratorios elaboradores de los 10 medicamentos más caros y más baratos

- **Laboratorios de los medicamentos más caros:** Los laboratorios asociados a los medicamentos más caros son principalmente grandes farmacéuticas, muchas de las cuales son líderes en investigación y desarrollo, lo que puede influir en el precio de sus productos.
- **Laboratorios de los medicamentos más baratos:** Los laboratorios que producen los medicamentos más baratos también son variados, incluyendo tanto grandes como pequeñas empresas, y en muchos casos se corresponden con productores de genéricos o medicamentos con mayor competencia en el mercado.

### 7. Valor promedio del costo de los medicamentos

- **Costo promedio general:** El valor promedio del COPAGO de los medicamentos analizados es de $14435.03. Este promedio es influenciado por la gran dispersión en los precios, donde pocos medicamentos de muy alto costo pueden elevar significativamente la media, mientras que una gran cantidad de medicamentos de bajo costo tiran el promedio hacia abajo.

### Observaciones Adicionales:

- La diferencia porcentual del COPAGO entre el laboratorio más caro y el más barato es notablemente alta, indicando una gran variabilidad en los costos dependiendo del laboratorio y el tipo de medicamento.
- Existe una relación inversa esperada entre la cobertura y el COPAGO, donde a mayor cobertura de PAMI, menor es el COPAGO para el afiliado, aunque la distribución muestra que incluso con altas coberturas, algunos medicamentos pueden tener un COPAGO considerable debido a su precio base.

En resumen, PAMI ofrece un amplio rango de coberturas, desde el 100% para muchos medicamentos y accesorios destinados al tratamiento convencional de la diabetes hasta coberturas parciales que van del 40 al 80% para tratamientos más especializados y costosos. Los medicamentos de alto costo están asociados a principios activos específicos y de reciente desarrollo. Es el caso de la semaglutida, también utilizado para la diabetes y obesidad.